client/src/document/ingredients/browser-compatibility-table/atoms/_bc-history.scss

```diff
@@ -38,6 +38,10 @@
 .bc-history-mobile {
   display: block;
 
+  dl {
+    padding: 10px;
```

Comment:
@schalkneethling 
Here, can you figure out why this isn't taking effect?
It's what I pinged you, on Slack, about for a Zoom debug chat.

Reply:
Oh, that is because `.bc-history-mobile` is the `dl` so, this will work:

```
.bc-history-mobile {
  display: block;
  padding: $base-spacing / 2;
}
```

Reply:
I am going to merge this one as is and open a quick self-merging-pr to update that.